Doc/library/stdtypes.rst

```diff
@@ -4106,8 +4106,10 @@ its contents cannot be altered after it is created; it can therefore be used as
 a dictionary key or as an element of another set.
 
 Non-empty sets (not frozensets) can be created by placing a comma-separated list
-of elements within braces, for example: ``{'jack', 'sjoerd'}``, in addition to the
-:class:`set` constructor.
+of elements within braces, for example: ``{'jack', 'sjoerd'}``.  Sets can
+be created by using a set comprehension, for example ``{2 * x for x in iterable}``.
+Also, set comprehensions can make both empty and non-empty sets.  In addition to
+the :class:`set` constructor.
```

Comment:
```suggestion
of elements within braces, for example ``{'jack', 'sjoerd'}``.  Sets can
be created by using a :term:`set comprehension`, for example ``{2 * x for x in iterable}``,
or using the :class:`set` constructor.
```
- Remove unnecessary sentence about creating empty sets using a setcomp.
- Fix incomplete sentence
- Link to the glossary